src/main/java/com/exasol/repository/ReleaseChangesLetter.java

```diff
@@ -0,0 +1,164 @@
+package com.exasol.repository;
+
+import static com.exasol.ReleaseRobotConstants.*;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * This class represents a changes file's content.
+ */
+public class ReleaseChangesLetter {
```

Comment:
Just call it "ReleaseLetter" :-)

Reply:
Okay, but it's not really a release letter. The release letter is inside of this file, as I see it...